packages/client/ui/react-ui/src/components/auth/AuthForm.tsx

```diff
@@ -8,7 +8,7 @@ import { FarcasterProvider } from "../../providers/auth/FarcasterProvider";
 import { classNames } from "@/utils/classNames";
 
 export function AuthForm({ className }: { className?: string }) {
-    const { step, appearance, baseUrl, loginMethods } = useAuthForm();
+    const { step, appearance, loginMethods, baseUrl } = useAuthForm();
```

Comment:
Unnecessary?

Reply:
yeah, initially I removed baseUrl thinking I didn't need it, but it ended up being necessary in one place and I added it back and it stayed like this haha